### ISSUE TITLE
Do you have updated code files for teacher models or ToXCL files

### ISSUE THREAD

**Issue by Ronfhermions** [NhatHoang2002/ToXCL#3]
Do you have updated code files for teacher models or ToXCL files? I want to add module shortcuts and modify modules, but I cannot reflect the model files you gave me in the Python files for modification。
Excuse me, thank you

**Comment by NhatHoang2002** [comment]
Hi @Ronfhermions, I don't have updated files. Which part(s) of the code are hard to modify?

**Comment by Ronfhermions** [comment]
> Hi [@Ronfhermions](https://github.com/Ronfhermions), I don't have updated files. Which part(s) of the code are hard to modify?您好，我没有更新的文件。代码的哪些部分很难修改？

I would like to obtain the code for the Toxcl model that performs better, which you provided in the previous question.

**Comment by Ronfhermions** [comment]
Because it seems that I can't figure out any differences between those model files and the github project. Because the hyperparameters in the config.json file are the same as the results I obtained, but the results of the github project are lower than that of the model.

**Comment by NhatHoang2002** [comment]
Hi, sorry for the late reply. I didn’t change the hyperparameters of the Flan-T5 (in config.json) so you saw it is still the same. I meant the other hyperparameters in my train.py file like learning rate, weight loss contribution of the 3 losses, etc.
